test/src/e2e_vm_tests/test_programs/should_pass/language/primitive_type_argument/src/main.sw

```diff
@@ -0,0 +1,14 @@
+script; 
```

Comment:
Can we actually have a more elaborate test with multiple type arguments and maybe a longer call path?

Reply:
I've added some more thorough test cases, let me know what you think 😄

Reply:
Nice!